src/Web/Avalonia.Web.Blazor/ClipboardImpl.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Avalonia.Input;
+using Avalonia.Input.Platform;
+using Microsoft.JSInterop;
+
+namespace Avalonia.Web.Blazor
+{
+    internal class ClipboardImpl : IClipboard
+    {
+        public async Task<string> GetTextAsync()
+        {
+            return await AvaloniaLocator.Current.GetRequiredService<IJSInProcessRuntime>().
+                InvokeAsync<string>("navigator.clipboard.readText");
+        }
+
+        public async Task SetTextAsync(string text)
+        {
+            await AvaloniaLocator.Current.GetRequiredService<IJSInProcessRuntime>().
+                InvokeAsync<string>("navigator.clipboard.writeText",text);
+        }
+
+        public Task ClearAsync() => Task.CompletedTask;
```

Comment:
Should be sufficient to send an empty string to a clipboard, if there is no better API

Reply:
I was thinking the same thing, but I wasn't sure. I can only find 4 methods for the api, nothing has to do with clearing and the read(), write() are only used to copy within the browser sandbox, so not really useful.